typescript/common-resolvers/src/export/index.ts

```diff
@@ -48,7 +48,7 @@ const exportDataset = async (
       origin
     )
   )?.uploadUrl;
-  await repository.upload.put(outUrl, fileExport);
+  await repository.upload.put(outUrl, fileExport, req);
```

Comment:
I can't see any put function in any of the three repos that needs `req`, why was this added?

Reply:
Yes, that's a remnant of me trying to "fetch" regularly for Supabase: you needed to add the credentials in the request since they are checked when uploading on the upload route. I agree that I don't use them but in the end I felt like it could be useful in the future plus it's more symmetric with `repository.upload.get` that needs `req` for the same reason. What do you think?

Reply:
Ok let's leave it then, I agree it could be useful in the future